backend/src/interfaces/Ientities/IclientCourses.d.ts

```diff
@@ -33,7 +33,8 @@ interface IClientCourse {
   ];
   date: Date;
   applyDate?: Date;
-  testDate?: Date;
```

Comment:
if you replace testDate please also replace all connected data and fns with testDate on frontend  according to new logic

Reply:
done